### PR TITLE
Acm Promotion module D9 ready by removing deprecated code/dependency fix etc.

### PR DESCRIPTION
**Pull request by joshirohit100**

### CODE REVIEW

File: modules/acm_promotion/src/Plugin/Block/AcmPromotionBlock.php [hunk 1]
```diff
@@ -159,19 +204,20 @@ public function build() {
    *
    * This method loads the active promotion nodes.
    *
-   * @return Drupal\node\Entity\Node[]
+   * @return \Drupal\node\Entity\Node[]
```
Comment (miromichalicka) [inline modules/acm_promotion/src/Plugin/Block/AcmPromotionBlock.php]: Should this be `NodeInterface[]` ?

Comment (joshirohit100) [reply modules/acm_promotion/src/Plugin/Block/AcmPromotionBlock.php]: Yes! as per standard, type should be interface. I only fixed the errors thrown by deprecation check. I will update to use entity interface

Comment (joshirohit100) [reply modules/acm_promotion/src/Plugin/Block/AcmPromotionBlock.php]: PR updated! Thanks!

**Comment by joshirohit100** [comment]
Travis failure does not look related to this.

**Pull request merged**